src/router.js

```diff
@@ -19,11 +20,15 @@ export const ROUTE_NAME_SIGNUP = 'Signup';
 export const ROUTE_NAME_STATS = 'Stats';
 export const ROUTE_NAME_STATS_SEASON = 'StatsBySeason';
 
-const mustBeAuthenticated = (to, from, next) => {
+const mustBeAuthenticated = async (to, from, next) => {
```

Comment:
I don't think this needs to be async as the action itself is only making local storage calls which is synchronous

```suggestion
const mustBeAuthenticated = (to, from, next) => {
```